react/features/base/lib-jitsi-meet/middleware.ts

```diff
@@ -94,7 +94,6 @@ function _setErrorHandlers() {
 
         // TODO: Don't remove this ignore. The build fails on macOS and we don't know yet why.
 
-        // @ts-ignore
```

Comment:
If this works now, pl remove the comment above.

Reply:
It is needed for `ts-loader`